### PR TITLE
feat: Updates with PG verified recommendations and upgraded 3 recommendations to impact to High

### PR DESCRIPTION
**Pull request by aarthiem**
# Overview/Summary

PG verified on  8 recommendations and below recommendations impact is set to High
--Enable Health check for App Services
--Set minimum instance count to 2 for app service (note that this currently needs to be set to 3 for zone redundancy to be enabled, but will be lowered to 2 by end of year).
--Monitor Performance

### Breaking Changes

1. _Replace me_
2. _Replace me_

## As part of this pull request I have

- [x ] Read the [Contribution Guide](https://azure.github.io/Azure-Proactive-Resiliency-Library-v2/contributing) and ensured this PR is compliant with the guide
- [x ] Checked for duplicate [Pull Requests](https://github.com/Azure/Azure-Proactive-Resiliency-Library-v2/pulls)
- [ ] Associated it with relevant [GitHub Issues](https://github.com/Azure/Azure-Proactive-Resiliency-Library-v2/issues) or ADO Work Items (Internal Only)
- [x ] Ensured my code/branch is up-to-date with the latest changes in the `main` [branch](https://github.com/Azure/Azure-Proactive-Resiliency-Library-v2/tree/main)
- [x ] Ensured PR tests are passing
- [ ] Performed testing and provided evidence (e.g. screenshot of output) for any changes associated to ARG queries
- [ ] Updated relevant and associated documentation (e.g. Contribution Guide, Docs etc.)

### CODE REVIEW

**Review by seligj95 — APPROVED** [review]

**Review by oZakari — APPROVED** [review]
LGTM

File: azure-resources/Web/sites/recommendations.yaml [hunk 1]
```diff
@@ -112,14 +112,14 @@
...
-  recommendationImpact: Medium
+  recommendationImpact: High
   recommendationResourceType: Microsoft.Web/sites
   recommendationMetadataState: Active
   longDescription: |
     Use Health Check for production workloads. Health check increases your application's availability by rerouting requests away from unhealthy instances, and replacing instances if they remain unhealthy. The Health check path should check critical components of your application.
   potentialBenefits: Enhanced reliability via automation
   pgVerified: false
-  publishedToLearn: false
+  publishedToLearn: true
```
Comment (ejhenry) [inline azure-resources/Web/sites/recommendations.yaml:122]: @aarthiem did you mean to change the pgVerified on line 121?

File: azure-resources/Web/sites/recommendations.yaml [hunk 2]
```diff
@@ -112,14 +112,14 @@
...
   recommendationTypeId: null
   recommendationControl: Other Best Practices
-  recommendationImpact: Medium
+  recommendationImpact: High
   recommendationResourceType: Microsoft.Web/sites
   recommendationMetadataState: Active
   longDescription: |
     Use Health Check for production workloads. Health check increases your application's availability by rerouting requests away from unhealthy instances, and replacing instances if they remain unhealthy. The Health check path should check critical components of your application.
   potentialBenefits: Enhanced reliability via automation
   pgVerified: false
```
Comment (ejhenry) [inline azure-resources/Web/sites/recommendations.yaml:121]:
```suggestion
  pgVerified: true
```

File: azure-resources/Web/sites/recommendations.yaml (hunk 1 above)
Comment (ejhenry) [inline azure-resources/Web/sites/recommendations.yaml:122]:
```suggestion
  publishedToLearn: false
```

**Review by ejhenry — APPROVED** [review]

**Review by oZakari — APPROVED** [review]

**Pull request merged**